e2e/test/iothub/method/MethodE2ETests.cs

```diff
@@ -301,6 +303,65 @@ await deviceClient
             }
         }
 
+        [TestMethod]
+        [Timeout(TestTimeoutMilliseconds)]
+        public async Task Method_ServiceInvokeDeviceMethodWithDateTimePayload_DoesNotThrow()
+        {
+            // arrange
+
```

Comment:
nit: extra line

Reply:
this extra line is actually left intentionally as we have a code block below other than a single line in the "arrange" section.